app/models/shelter.rb

```diff
@@ -1,3 +1,3 @@
 class Shelter < ApplicationRecord
-  has_many :pets
+  has_many :pets, dependent: :destroy
```

Comment:
was getting the following error: `ActiveRecord::InvalidForeignKey:
       PG::ForeignKeyViolation: ERROR:  update or delete on table "shelters" violates foreign key constraint "fk_rails_92fb5d7a05" on
 table "pets"
       DETAIL:  Key (id)=(721) is still referenced from table "pets".
       : DELETE FROM "shelters" WHERE "shelters"."id" = $1`
added `dependent: :destroy` to address the error. This also removes any associated records in the pets table.